cluster-service/deploy/helm/templates/deployment.yaml

```diff
@@ -0,0 +1,215 @@
+        - name: mixin-pull-secret
+          mountPath: /secrets/mixin-pull-secret
+        - name: instance-types
+          mountPath: /configs/cloud-resources/instance-types.yaml
+          subPath: instance-types.yaml
+        - name: instance-type-constraints
+          mountPath: /configs/cloud-resource-constraints/instance-type-constraints.yaml
+          subPath: instance-type-constraints.yaml
+        - name: cloud-regions
+          mountPath: /configs/cloud-resources/cloud-regions.yaml
+          subPath: cloud-regions.yaml
+        - name: cloud-region-constraints
+          mountPath: /configs/cloud-resource-constraints/cloud-region-constraints.yaml
+          subPath: cloud-region-constraints.yaml
+        - name: azure-credentials
+          mountPath: /secrets/azure-credentials
+        - name: keyvault
+          mountPath: "/secrets/keyvault"
+          readOnly: true
+        - name: azure-runtime-config
+          mountPath: /configs/azure-runtime-config
+        - name: azure-operators-managed-identities-config
+          mountPath: /configs/azure-operators-managed-identities-config.yaml
+          subPath: azure-operators-managed-identities-config.yaml
+        command:
+        - /usr/local/bin/clusters-service
+        - serve
+        - --log-level={{ .Values.logLevel }}
+        - --namespace={{ .Release.Namespace }}
+        - --runtime-mode={{ .Values.runtimeMode }}
+        - --default-expiration={{ .Values.defaultExpiration }}
+        - --maximum-expiration={{ .Values.maximumExpiration }}
+        - --db-host=@/secrets/rds/db.host
+        - --db-port=@/secrets/rds/db.port
+        - --db-name=@/secrets/rds/db.name
+        - --db-user=@/secrets/rds/db.user
+        - --db-password=@/secrets/rds/db.password
+        - --db-disable-tls={{ .Values.databaseDisableTls }}
+        - --db-auth-method={{ .Values.databaseAuthMethod }}
+        - --gateway-url={{ .Values.gatewayURL }}
+        - --client-id=@/secrets/service/client.id
+        - --client-secret=@/secrets/service/client.secret
+        - --client-scopes={{ .Values.clientScopes }}
+        - --user-defined-dns-base-domain={{ .Values.userDefinedDnsBaseDomain }}
+        - --jwks-url={{ .Values.jwksUrl }}
+        - --jwks-file=/configs/authentication/jwks.json
+        - --acl-file=/configs/authentication/acl.yml
+        - --token-url={{ .Values.tokenUrl }}
+        - --insecure={{ .Values.insecure }}
+        - --api-listener-network=tcp
+        - --api-listener-address=:8000
+        - --metrics-listener-network=tcp
+        - --metrics-listener-address=:8080
+        - --healthcheck-listener-network=tcp
+        - --healthcheck-listener-address=:8083
+        - --environment={{ .Values.environment }}
+        - --backplane-url={{ .Values.backplaneURL }}
+        - --provision-shards-config=/secrets/shards/config
+        - --proxy-config-file=/configs/proxy/config.yaml
+        - --aws-sts-policy-directory=/configs/policies
```

Comment:
This flag isn't applicable for aro-hcp.
We can clean up later;
At the moment if we don't define it, CS starts normally but we'll get a 
```
2024-12-12T15:00:50Z ERROR loader/aws_sts_policy_loader.go:50 [opid='dc2b7851-64e2-4869-b48a-d4a21c5a407a'] Got error while trying to load policies: open : no such file or directory
gitlab.cee.redhat.com/service/uhc-clusters-service/pkg/aws/loader.(*AWSPolicyLoader).run
	/cs/pkg/aws/loader/aws_sts_policy_loader.go:50
gitlab.cee.redhat.com/service/uhc-clusters-service/pkg/aws/loader.(*AWSPolicyLoader).Start
	cs/pkg/aws/loader/aws_sts_policy_loader.go:43
gitlab.cee.redhat.com/service/uhc-clusters-service/cmd/clusters-service/servecmd.runServ
```
error message at start time, which can be confusing.